src/pylptlib/lpt.py

```diff
@@ -889,3 +947,330 @@ def decode_pulse_status(status_bits: int) -> dict:
     status["Measurement type"] = meas_types[status["Measurement type"]]
 
     return status
+
+
+# ### CVU
+def rdelay(delay: float):

```

Comment:
Is 'rdelay' maybe a more general command and would better fit to another section instead of CVU?